docs: add instructions for using AsyncAPI CLI (#433)
asyncapi/html-template: README.md

<!-- toc -->

- [Usage](#usage)

>   - [Using Generator](#using-generator)

Reviewer: you can remove that section entirely, we should promote AsyncAPI CLI only

- [Usage](#usage)
- [Supported parameters](#supported-parameters)
- [Development](#development)
- [Contributors ✨](#contributors-)

<!-- tocstop -->

## Usage

Install AsyncAPI CLI

```bash
npm install -g @asyncapi/cli
```

Generate using CLI

```bash

> asyncapi generate fromTemplate <asyncapi.yaml> @asyncapi/html-template@0.16.0

Reviewer: ```suggestion
asyncapi generate fromTemplate <asyncapi.yaml> @asyncapi/html-template@0.28.0
```

- [Usage](#usage)
- [Supported parameters](#supported-parameters)
- [Development](#development)
- [Contributors ✨](#contributors-)

<!-- tocstop -->

## Usage

Install AsyncAPI CLI

```bash
npm install -g @asyncapi/cli
```

Generate using CLI

```bash
asyncapi generate fromTemplate <asyncapi.yaml> @asyncapi/html-template@0.28.0
```

> You can replace <asyncapi.yaml> with local path or URL pointing to [any AsyncAPI document](https://raw.githubusercontent.com/asyncapi/spec/master/examples/streetlights-kafka.yml).

Reviewer: ```suggestion
You can replace `<asyncapi.yaml>` with local path or URL pointing to [any AsyncAPI document](https://raw.githubusercontent.com/asyncapi/spec/master/examples/streetlights-kafka.yml).
```

we should wrap such things in ` so they render in different way, as code

> To use a specific version, look into [Releases](https://github.com/asyncapi/html-template/releases).

Reviewer: ```suggestion
Look into [Releases](/releases) of this template to pick up the version you need. It is not recommended to always use latest in production. Always use specific version.
```

relative link should do the trick

> Look into [Releases](/releases) of this template to pick up the version you need. It is not recommended to always use the latest in production. Always use a specific version.

Reviewer: ```suggestion
Look into [Releases](/asyncapi/html-template/releases) of this template to pick up the version you need. It is not recommended to always use the latest in production. Always use a specific version.
```

sorry, I was wrong, tested now and the relative link must be different, the one that I suggest now

npm install -g @asyncapi/generator
```

### or 

### Using AsyncAPI CLI

Install template

```bash

> npm install -g @asyncapi/html-template@0.16.0

Reviewer: you do not have to install the template of you want to use it, the generator will install it anyway after running `asyncapi generate fromTemplate`.

you do `npm install -g @asyncapi/html-template@0.16.0` only in production, if you first want to make sure the specific version of the template is installed

you could also do `asyncapi generate fromTemplate asyncapi.yaml @asyncapi/html-template@0.16.0` and it should work

```

Install AsyncAPI CLI

```bash
npm install -g @asyncapi/cli
```

Generate using CLI

```bash

> asyncapi generate fromTemplate asyncapi.yaml @asyncapi/html-template

Reviewer: if you copy paste this command, it will not work for you right? so maybe workaround it by having a link to `asyncapi.yaml` or add proper explanation

possible link to use could be https://raw.githubusercontent.com/asyncapi/spec/master/examples/streetlights-kafka.yml
